services/app-api/forms/2026/measureTemplates.ts

```diff
@@ -448,6 +448,76 @@ export const measureTemplates: Record<
       measureFooter,
     ],
   },
+  [MeasureTemplateName["FFS-FASI-1"]]: {
+    id: "FFS-FASI-1",
+    title: "FASI-1: Fee-For-Service Measure Results",
+    type: PageType.MeasureResults,
+    sidebar: false,
+    elements: [
+      {
+        type: ElementType.ButtonLink,
+        id: "return-button",
+        label: "Return to Measure Information",
+        to: "FASI-1",
+      },
+      {
+        ...measureHeader,
+        text: "FASI-1: Fee-For-Service Measure Results",
+      },
+      feeForServiceMeasureResultsSubheader,
+      whichMedicaidHCBSprograms,
+      {
+        type: ElementType.NestedHeading,
+        id: "measure-subheader",
+        text: "Performance Rate: Participant who has Identified at Least as Many Total Personal Priorities as Functional Needs in the Areas of Self-Care, Mobility, or IADL",
+      },
+      performanceRateTarget,
+      performanceRateNum,
+      performanceRatesDenomAutoCalculates,
+      performanceRateAutoCalculates,
+      {
+        type: ElementType.MeasureFooter,
+        id: "measure-footer",
+        prevTo: "FASI-1",
+        completeSection: true,
+      },
+    ],
+  },
+  [MeasureTemplateName["MLTSS-FASI-1"]]: {
+    id: "MLTSS-FASI-1",
+    title: "FASI-1: Managed Care Measure Results",
```

Comment:
Similar to LTSS title, this one will look as such:
"FASI-1: Managed Care (MLTSS)"